src/topics/types.ts

```diff
@@ -0,0 +1,25 @@
+import { ICourierPaging } from "../types";
+
+export interface ICourierTopic {
+  created: number;
+  id: string;
+  name: string;
+  updated: number;
```

Comment:
`updated` is optional - when creating a topic, you only receive the following:

![image](https://user-images.githubusercontent.com/266481/90166717-83b42e80-dd4f-11ea-912e-26d8f43733ea.png)

Reply:
Cool. Thanks. I'm sure that would have been discovered once I started working on the update.